.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+name: CI
```

Comment:
Let's use the former workflow. It doesn't need to change much. We only need one Scala version (we can drop support for the others), and a test job can be added to the workflow.